src/quo2/components/messages_gap.cljs

```diff
@@ -0,0 +1,246 @@
+(ns quo2.components.messages-gap
+  (:require
+   ["react-native-svg" :as Svg]
+   [oops.core :refer [oget]]
+   [quo.react-native :as rn]
+   [quo.theme :as theme]
+   [quo2.components.text :as text]
+   [quo2.foundations.colors :as colors]
+   [re-frame.core :as re-frame]
+   [reagent.core :as reagent]
+   [status-im.i18n.i18n :as i18n]
+   [status-im.ui.components.react :refer [pressable-class]]))
+
+;;; helpers
+;; computed from
+;; box-shadow "inset 0px 0px 4px rgba(9, 16, 28, 0.08)"
+;; on to background-color  colors/neutral-5
+(defonce comupted-shadow-outer-color "#edeef0")
```

Comment:
what if the background color will change later by another dev?

Reply:
Currently, dev have to compute it manually. That's indeed prone to visual bug once there's universal color changes.
I'll add a util fn for calculating the computed result color of an alpha one above a background.

Reply:
Just managed to remove the computed color by stacking two layers (same shape, different color)